docs/permissions.rst

```diff
@@ -744,60 +744,71 @@ are as follows:
+    /users/example-user/resources/service-1/permissions                     => [write]  (1)
     /users/example-user/resources/service-2/permissions                     => []
-    /users/example-user/resources/resource-A/permissions                    => [read]
-    /users/example-user/resources/service-3/permissions                     => [write]
+    /users/example-user/resources/resource-A/permissions                    => [read]   (1)
+    /users/example-user/resources/service-3/permissions                     => [write]  (1)
     /users/example-user/resources/resource-B1/permissions                   => []
     /users/example-user/resources/resource-B2/permissions                   => []
 
-Using ``inherited`` option, we obtain the following:
+These simply represent the |direct_permissions|_ that ``example-user`` has, **without** considering all
+its :term:`Group` memberships.
+
+Using ``inherited=true`` option, we obtain the following:
 
 .. code-block::
 
     /users/example-user/resources/service-1/permissions?inherited=true      => [write]
-    /users/example-user/resources/service-2/permissions?inherited=true      => [write]  (1)
+    /users/example-user/resources/service-2/permissions?inherited=true      => [write]  (2)
     /users/example-user/resources/resource-A/permissions?inherited=true     => [read]
     /users/example-user/resources/service-3/permissions?inherited=true      => [write]
-    /users/example-user/resources/resource-B1/permissions?inherited=true    => [read]   (1)
+    /users/example-user/resources/resource-B1/permissions?inherited=true    => [read]   (2)
     /users/example-user/resources/resource-B2/permissions?inherited=true    => []
 
 As illustrated, requesting for |inherited_permissions|_ now also returns :term:`Group`-related :term:`Permission`
-:sup:`(1)` where they where not returned before with only :term:`User`-related :term:`Permission`.
+:sup:`(1)` that where not returned before with only :term:`User`-related :term:`Permission`. Note that returned
+:term:`Permissions <Permission>` are all combinations of |applied_permissions|_ originally defined.
 
-On the other hand, using ``effective`` would result in the following:
+On the other hand, using ``effective=true`` would result in the following:
 
 .. code-block::
 
-    /users/example-user/resources/service-1/permissions?effective=true      => [write]
+    /users/example-user/resources/service-1/permissions?effective=true      => [write]          (1)
     /users/example-user/resources/service-2/permissions?effective=true      => [write]          (2)
     /users/example-user/resources/resource-A/permissions?effective=true     => [read, write]    (3)
-    /users/example-user/resources/service-3/permissions?effective=true      => []
+    /users/example-user/resources/service-3/permissions?effective=true      => [write]          (1)
     /users/example-user/resources/resource-B1/permissions?effective=true    => [read]           (2)
```

Comment:
Doesn't `resource-B1` also inherit the `write` permission from the `service-3`, like the `resource-B2`?

Reply:
You're right!